packages/cfd/src/sass/cfd-dashboard.scss

```diff
@@ -1520,6 +1520,9 @@
         text-align: left;
         margin: 0 2.4rem;
     }
+    &__acc_status_badge {
+        max-width: fit-content;
```

Comment:
Can we use max-width as `max-content` to be more clear? https://www.quirksmode.org/blog/archives/2021/04/fitcontent_and.html